client/src/App.js

```diff
@@ -1,9 +1,11 @@
 import React, {Component} from 'react';
 import './App.css';
+import AppOfTheWeek from './Clusters/AppOfTheWeek/AppOfTheWeek.js';
+import Footer from './Clusters/Footer/Footer.js';
 import Header from './Header/header.js';
-import TopCharts from './Clusters/TopCharts/TopCharts.js';
+import RecommendedCategories from './Clusters/RecommendedCategories/recommended_categories.js';
```

Comment:
Did you accidentally remove RecommendedCategories?

Reply:
I actually don't see RecommendedCategories in the master branch code, should I add the component in this PR or I think essence is doing a PR with RecommendedCategories right now, should I leave it as is?

Reply:
Ah looks like Ayo accidentally deleted it in https://github.com/googleinterns/play-web-step-2020/pull/69/files

Can you add it back?